packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -1035,7 +1035,7 @@ class ReviewSectionComp extends React.Component<FullProps, State> {
               field,
               errorsOnFields,
               undefined,
-              !!index
+              !index
```

Comment:
The condition was flipped. Should be true for the first index and false otherwise